README.md

```diff
@@ -1,11 +1,10 @@
 ## What is Nebula?
 Nebula is a scalable overlay networking tool with a focus on performance, simplicity and security.
-It lets you seamlessly connect computers anywhere in the world. Nebula is portable, and runs on Linux, OSX, and Windows.
+The networking tool allows seamless connection of computers globally. Nebula is portable, and runs on Linux, OSX, and Windows.
```

Comment:
>  Pronouns confuse the reader because they do not know what the pronouns are referring to.

IMHO it seems obvious `It` refers to the Nebula tool as per the heading "What is Nebula" and the first sentence describing the tool. I don't think there is anything clarified by this change. 

> allows seamless **connection of**

This phrasing does not sound correct. Perhaps instead, if this must change:  "allows for a seamless connection of computers", or "allows you to seamlessly connect computers" ?